packages/langium/src/lsp/normalized-text-documents.ts

```diff
@@ -142,8 +153,28 @@ export class NormalizedTextDocuments<T extends { uri: string }> implements TextD
         return this._onDidClose.event;
     }
 
-    public get(uri: string): T | undefined {
-        return this._syncedDocuments.get(UriUtils.normalize(uri));
+    public get(uri: string | URI): T | undefined {
+        return this._syncedDocuments.get(UriUtils.normalize(uri.toString()));
```

Comment:
It's just a detail: the `toString()` is not required here considering that `normalize` accepts `URI|string`... 😉.